.github/buildomat/jobs/opte-api.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#:
+#: name = "opte-api"
+#: variety = "basic"
+#: target = "helios"
+#: rust_toolchain = "nightly"
+#: output_rules = []
+#:
+
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+function header {
+	echo "# ==== $* ==== #"
+}
+
+cargo --version
+rustc --version
+
+cd opte-api
+
+header "check API_VERSION"
+./check-api-version.sh
+
+header "check style"
+ptime -m cargo fmt -- --check
+
+header "analyze std"
+ptime -m cargo check
```

Comment:
Any particular reason we're checking here, if we're going to build in the next step? I usually think of `cargo check` as a quicker approximation of `cargo build`.

Reply:
You know that's a good question. I must of somehow got it in my head that `check` was doing additional stuff. But reading the cargo docs makes it clear it's the same as `build` without the final codegen. So we should remove these.

Reply:
I'm going to modify the build scripts in a separate PR just to keep from adding to this one.